packages/realm-app-importer/src/node/index.ts

```diff
@@ -0,0 +1,19 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2020 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Is this license note true?  Is Realm still under Apache License, or did we move to MongoDB's amended version?

Reply:
We're under Apache, but the copyright should probably say 2022.

Reply:
I've noticed that the year was off in several of the headers.  We should probably clean that up.